src/styles/main.scss

```
@@ -1,3 +1,9 @@
-body {
-  margin: 0;
-}
+@import "reset";
+@import "variables";
+@import "nav";
+@import "header";
```

```suggestion
@import "header";
```
в блоків (.header) не повинно бути margin-ів